virttest/libvirt_vm.py

```diff
@@ -1873,6 +1874,10 @@ def create(self, name=None, params=None, root_dir=None, timeout=5.0,
         params = self.params
         root_dir = self.root_dir
 
+        if self.params.get("storage_type") == "nfs":
+            image_name_only = os.path.basename(self.params["image_name"])
```

Comment:
this will be simplified too, if below change works,  we need not pass `image_name` here